index.php

```diff
@@ -125,30 +126,44 @@ function myErrorHandler($errno, $errstr, $errfile, $errline)
 
 $PHPErrors = '';
 
-// Read Config and Definitionfiles
-// Load Basic Config
-if (file_exists('inc/base/config.php')) {
-    $config = parse_ini_file('inc/base/config.php', 1);
+// Initialize Cache. Go for APCu first, filebased otherwise. DB adaptor to be used when we implement PDO.
```

Comment:
`DB adaptor` => `DB adapter`

Reply:
"adaptor" is correct British English (but also "adapter")